### PR TITLE
Account Module: Minor adjustment

### PR DESCRIPTION
**Pull request by andy-hook**
Address the issue we discussed with "Unknown" wallet during connection. Couple of other minor tweaks to click targets and header spacing while I was in the area.

### CODE REVIEW

File: src/components/Account/AccountModule.js [hunk 1]
```diff
@@ -28,20 +28,13 @@ function AccountModule() {
   const toggle = useCallback(() => setOpened((opened) => !opened), [])
 
   useEffect(() => {
-    let timer
```
Comment (delfipolito) [inline src/components/Account/AccountModule.js:31]: Just curious, why are we deleting this?

**Review by delfipolito — APPROVED** [review]
Looks good to me!

File: src/components/Account/AccountModule.js (hunk 1 above)
Comment (andy-hook) [reply src/components/Account/AccountModule.js:31]: This was added as workaround for an animation issue [here](https://github.com/aragon/court-dashboard/pull/375/files#r463716108) and setting this value to `null` after the timeout is what makes the module unable to determine the type of wallet that is connecting. I gave it a good amount of testing and saw no animation issues without it.

Comment (delfipolito) [reply src/components/Account/AccountModule.js:31]: awesome! :)

Comment (andy-hook) [reply src/components/Account/AccountModule.js:31]: But keep an eye out, if all is well we can suggest the same changes in our other repos :)

**Pull request merged**